### ISSUE TITLE
Gow ghost of sparta bugs and glitches

### ISSUE THREAD

**Issue by Knownnonymous** [hrydgard/ppsspp#18612]
### Game or games this happens in

Gow ghost of sparta

### What area of the game / PPSSPP

So the point is the problems started with the nexus of Atlantis, where the game crashes but it just solved when someone turn off the fast memory on ppsspp, I used v1.16.6 gold mod apk😅

But I faced a lots of bugs other than this... The first one is when you use the athena's fire blade by pressing R is kinda rough swinging like I cannot see the smoothness like the normal blade, and when we enable the fast memory and only use the fire blades by R button then there's sudden fps drop 2-3fps/60

And the second one is the death gate when I enter the screen just turned white.. And there's no touch nor any sound after that.. Also I have 3 save states and one of them crashes with blue screen after i open with fast memory or enabled in game.. And unfortunately I just deleted everything related to Gow.. Just want to ask if i install the latest git build of ppsspp, isn't there any bugs or glitches??cuz again I don't want this to happen again.. 

(NOTE : all these problems only happened when you cross the nexus of atlantis) 

### What should happen

The game shouldn't be crashed by blue screen, and also the death gate entering shouldn't be a problem.. the nexus of atlantis issue should be fix.. I really dont have any video or photo where I can show the issues.. 

### Platform

Android

### Mobile device model or graphics card (GPU)

Redmi 12 (5g) 

### PPSSPP version affected

Only showing v1.16.6

### Last working version

Don't know, will try the latest git build if assured there's no issue.

### Graphics backend (3D API)

Vulkan

### Checklist

- [ ] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [ ] Try without any cheats and without loading any save states.
- [ ] Include logs or screenshots of issue.

**Comment by anr2me** [comment]
Similar issue to this https://github.com/hrydgard/ppsspp/issues/14958
And it have a workaround to avoid the crash, by changing CPU clock to 200 i think, you can read more about it near the latest posts there.

**Comment by LunaMoo** [comment]
Preferably you should start by saving the game normally and reload from that in game save not using buggy savestates anymore as savestates not only keep most of the issues, they also simulate playing for a very long time and most games become unstable after enough hours even on real hardware.

**Comment by Knownnonymous** [comment]
Yeah, well alright.. Just going to do what you guys said :D

**Comment by davidhedlund** [comment]
Issue: See [my uploaded video](https://github.com/hrydgard/ppsspp/assets/6416073/04536eaf-24be-4e9a-a285-f14367e7c2fc) -- This is part 17 or 21 in the game
PPSSPP version: 1.16.6
OS: Android 10

Expected result: https://www.youtube.com/watch?v=jcIZFskcp3o&list=PLJw500QNRAdCNQgaF__PrmRVQ2Vych9AY&index=17&t=674s

**Comment by davidhedlund** [comment]
> ### Game or games this happens in
> 
> Gow ghost of sparta
> ### What area of the game / PPSSPP
> 
> So the point is the problems started with the nexus of Atlantis, where the game crashes but it just solved when someone turn off the fast memory on ppsspp

I can confirm this.

Issue: Nexus of Atlantis, see [my uploaded video](https://github.com/hrydgard/ppsspp/assets/6416073/04536eaf-24be-4e9a-a285-f14367e7c2fc)
Workaround: Settings -> Disable: Fast memory (unstable)
PPSSPP version: 1.16.6
OS: Android 10

**Comment by davidhedlund** [comment]
> > ### Game or games this happens in
> > Gow ghost of sparta
> > ### What area of the game / PPSSPP
> > So the point is the problems started with the nexus of Atlantis, where the game crashes but it just solved when someone turn off the fast memory on ppsspp
> 
> I can confirm this.
> 
> Issue: Nexus of Atlantis, see [my uploaded video](https://github.com/hrydgard/ppsspp/assets/6416073/04536eaf-24be-4e9a-a285-f14367e7c2fc) Workaround: Settings -> Disable: Fast memory (unstable) PPSSPP version: 1.16.6 OS: Android 10

Here is the save state file: [PPSSPP_STATE.zip](https://github.com/hrydgard/ppsspp/files/13785738/PPSSPP_STATE.zip)

Let me know if the game save file should also be uploaded. @hrydgard

**Comment by davidhedlund** [comment]
>     * Game ID: NPHG00092_1.00

Updates for https://report.ppsspp.org/games?name=ghost+of+sparta
* https://report.ppsspp.org/game/NPHG00092_1.00 should be downgraded to Playable instead of Perfect
* "[God of War™: Ghost of Sparta --NATIVE-DATA-BOUNDARY-6da69a33$:dbb062-1 Cowent-D�rposition: form-data; name="sdkver" Content-Length: 9 Content-Transfer-Encoding: binary 100859920 --NATIVE-DATA-BOUNDARY-6da69a�I:܍��̠���G܍��̠� ����`r](https://report.ppsspp.org/game/NPHG00092_1.00,NPH)" should be removed, this is just a duplicated and malformed report

**Comment by davidhedlund** [comment]
> So the point is the problems started with the nexus of Atlantis, where the game crashes but it just solved when someone turn off the fast memory on ppsspp

This is a duplicated issue of:
* https://github.com/hrydgard/ppsspp/issues/14958

**Comment by Knownnonymous** [comment]
That's not exact duplicate, i found new issues in game, that's why

**Comment by davidhedlund** [comment]
> That's not exact duplicate, i found new issues in game, that's why

Yes I know, that's great. I didn't mean that your whole report was a duplicate.

**Comment by Knownnonymous** [comment]
i beat the game, without fast memory, and no changes in clock speed..

Tip: don't save your game in ppsspp save states instead in game save and when you are jumping in "noa" you should backward a little and facing the left, then jump..

**Comment by davidhedlund** [comment]
> i beat the game, with fast memory checked, and no changes in clock speed..
> 
> Tip: don't save your game in ppsspp save states instead in game save and when you are jumping in "noa" you should backward a little and facing the left, then jump..

That's cool! You could mention this in https://github.com/hrydgard/ppsspp/issues/14958

However, I'd appreciate if you reopen the issue. You can just strikethrough the nexus of Atlantis issue in the top post and refer to https://github.com/hrydgard/ppsspp/issues/14958. As I agreed with you before, your other bug issues are still useful to fix for the game.

**Comment by Knownnonymous** [comment]
Donee!! Sir
